pages/common/pdftk.md

```diff
@@ -0,0 +1,23 @@
+# pdftk
+
+> PDF toolkit.
+
+- Extract range:
+
+`pdftk {{input.pdf}} cat {{1-3 5 6-10}} output {{output.pdf}}`
+
+- Merge PDF:
+
+`pdftk {{file1.pdf}} {{file2.pdf}} ... cat output {{output.pdf}}`
```

Comment:
@ostera I have done it as in https://github.com/tldr-pages/tldr/blob/master/pages/common/convert.md